src/i_oalmusic.c

```diff
@@ -458,13 +459,26 @@ static const char **I_OAL_DeviceList(void)
     return devices;
 }
 
+static midiplayertype_t I_OAL_MidiPlayerType(void)
+{
+#ifdef HAVE_FLUIDSYNTH
+    if (active_module == &stream_fl_module)
+    {
+        return midiplayer_fluidsynth;
+    }
+#endif
+    if (active_module == &stream_opl_module)
+    {
+        return midiplayer_opl;
+    }
+    return midiplayer_none;
+}
+
 static void I_OAL_BindVariables(void)
 {
-    BIND_NUM_MIDI(opl_gain, 200, 100, 1000,
-        "Fine tune OPL emulation output level (default 200%)");
+    BIND_NUM_MIDI(opl_gain, 0, -20, 20, "OPL emulation gain [dB]");
 #if defined (HAVE_FLUIDSYNTH)
-    BIND_NUM_MIDI(mus_gain, 100, 10, 1000,
-        "Fine tune FluidSynth output level (default 100%)");
+    BIND_NUM_MIDI(mus_gain, 0, -20, 20, "FluidSynth gain [dB]");
```

Comment:
Since we have changed the `mus_gain` values, perhaps we should rename the `mus_*` variables? I suggest `fl_gain`.

Reply:
If the config values are out of range, they reset to default values right? (e.g. old config "100" becomes "0") So it should be okay, but if you want to be safe, sure.

Reply:
Okay, it's fine as it is.